source/StarStreak.js

```diff
@@ -0,0 +1,31 @@
+var Pixi = require("pixi.js")
+
+import Reference from "./Reference.js"
+import Utility from "./Utility.js"
+
+export default class StarStreak extends Pixi.Sprite {
+    constructor(parallax) {
```

Comment:
It doesn't look like `parallax` is getting used in `StarStreak`. No sense passing it to the constructor if it's unneeded.